src/functionalTest/resources/features/F-205 - Validate NoC Questions/S-205.1.td.json

```diff
@@ -0,0 +1,35 @@
+{
+  "_guid_": "S-205.1",
+  "_extends_": "S-205_Test_Data_Base",
+  "specs" : [
+    "Richard - with the ability to create a case for a particular jurisdiction within an organisation",
+    "Dil - with a solicitor role for the same jurisdiction within a different organisation from Richard",
+    "the configuration for the case type of Case C1 is such that there is a NoCRequest event available to users with the IDAM ID Caseworker-CAA when the case is in the state established following case creation",
+    "is made by Dil to retrieve the NOC questions that must be answered in order to raise a NOC Request to become the representative for Mario on C1",
+    "is made by Dil to validate the answers provided to the NoC questions in order to become the representative for Mario on C1",
+    "contains all correct answers"
+
+  ],
+
+  "users": {
+    "invokingUser": {
+      "_extends_": "ACA_Users_Dil"
+    }
+  },
```

Comment:
If two users in specs list should have two users here (with order matching the order in feature file).